scripts/externalTests/common.sh

```diff
@@ -106,25 +106,28 @@ function setup_solc
 
 function download_project
 {
-    local repo="$1"
-    local ref_type="$2"
-    local solcjs_ref="$3"
+    local repo_url="$1"
+    local ref="$3"
     local test_dir="$4"
 
-    [[ $ref_type == commit || $ref_type == branch || $ref_type == tag ]] || assertFail
-
-    printLog "Cloning ${ref_type} ${solcjs_ref} of ${repo}..."
-    if [[ $ref_type == commit ]]; then
-        mkdir ext
-        cd ext
-        git init
-        git remote add origin "$repo"
-        git fetch --depth 1 origin "$solcjs_ref"
-        git reset --hard FETCH_HEAD
-    else
-        git clone --depth 1 "$repo" -b "$solcjs_ref" "$test_dir/ext"
-        cd ext
+    printLog "Cloning ${repo_url}..."
+    # Clone the repo ignoring all blobs until needed by git.
+    # This allows access to commit history but with a fast initial clone
+    git clone --filter=blob:none "$repo_url" "$test_dir/ext"
+    cd "$test_dir/ext"
+
+    # If the ref_type is 'release' try to use the latest tag as ref
+    # NOTE: Sadly this will not work with monorepos and may not always
+    # return the latest tag.
+    if [[ "$ref" == "<latest>" ]]; then
```

Comment:
If you're removing `ref_type`, I'd make this more specific to make it clear it's the latest release not e.g. the latest commit:
```suggestion
    if [[ "$ref" == "<latest_release>" ]]; then
```